### DISCUSSION TITLE
-x question

### DISCUSSION THREAD

**Discussion by bongobongoland** [Pennyw0rth/NetExec#18] [Q&A]
@XiaoliChan 
Can't figure this out.
how to I execute the following command (using -x or -X)

\\\\192.168.1.1\\share\\whoami.exe

?

I'm getting all sorts of errors. 

I've tried \\\\\\\192.168.1.1\\\\share\\\\whoami.exe
I get access denied.
Do I need to use " or ' or both?

**Comment by XiaoliChan** [comment]
How did you run the smbserver?

![image](https://github.com/Pennyw0rth/NetExec/assets/30458572/1e413a75-65d7-4e27-9451-e5f8f4303cac)

![image](https://github.com/Pennyw0rth/NetExec/assets/30458572/f0904010-c135-4a9c-a626-64dc61e82b4b)

Sometimes it may not work, but you can use `net use` to mount the shares. (please check more videos from ippsec)

**Comment by bongobongoland** [comment]
The issues is that I can access the share folder easily via PS or cmd. just running \\192.168.1.1\share\whoami.exe works fine. so the share is working fine. but if I try to use -x, I get access denied...

**Comment by XiaoliChan** [comment]
> The issues is that I can access the share folder easily via PS or cmd. just running \192.168.1.1\share\whoami.exe works fine. so the share is working fine. but if I try to use -x, I get access denied...

Maybe got AV detection?

I always don't use this technique

**Comment by bongobongoland** [comment]
AV? I guess that's possible!

**Reply by XiaoliChan** [reply]
Yes, UNC has already mark as high risky many years ago, I can tell you the defender can do scanning on your shares when you access it.
